heuristic/repair_operators/greedy_insert.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from numpy.random import RandomState
+import heapq
+
+
+from heuristic.classes import Item, Route, Solution, Stacks
+from heuristic.constants import DEPOT
+
+
+def greedy_insert(current: Solution, rnd_state: RandomState) -> Solution:
+    """
+    TODO.
```

Comment:
And add a description. This is related to `sequential best insertion` in Hornstra et al. (2020).

Reply:
Including a run-time complexity!